update-github-pages.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
 cd "$HOME"
+
+git config credential.helper "store --file=.git/credentials"
+echo "https://${GITHUB_TOKEN}:@github.com" > .git/credentials
```

Comment:
I was just told the colon was right. They separate username from password and an empty one indicates git to use the "no password" authentication.